transactions/btc.ts

```diff
@@ -65,6 +65,15 @@ export function selectUnspentOutputs(
     sumValue += pinnedOutput.value;
   }
 
+  // Sort UTXOs based on block time in ascending order
+  unspentOutputs.sort((a, b) => {
```

Comment:
we can sort and calculate the sum in one `Arr.reduce` statement instead of looping twice here

Reply:
Yeah, I don't think that's possible 👀  We'd need to select from the sorted list.